src/components/ReviewConfirm/IncorporationDateTime.vue

```diff
@@ -0,0 +1,368 @@
+        <label>Incorporation Date and Time</label>
+      </v-col>
+      <v-col cols="12" sm="8" md="7">
+        <v-radio-group
+          column
+          class="radio-group"
+          v-model="selectDate">
+          <v-radio
+            label="Immediate (Upon Filing)"
+            value="isImmediate">
+          </v-radio>
+          <v-radio
+            label="A date / time in the future"
+            value="isFutureEffective">
+          </v-radio>
+        </v-radio-group>
+        <div class="date-time-selectors">
+          <v-menu
+            :close-on-content-click="false"
+            :nudge-right="40"
+            transition="scale-transition"
+            offset-y
+            min-width="290px"
+          >
+            <template v-slot:activator="{ on }">
+              <v-text-field
+                id="date-text-field"
+                placeholder="Date"
+                append-icon="mdi-calendar"
+                v-model="dateText"
+                :disabled="!isFutureEffective"
+                v-on="on"
+                readonly
+                filled
+              />
+            </template>
+            <v-date-picker
+              v-model="datePicker"
+              :min=minDate
+              :max=maxDate>
+            </v-date-picker>
+          </v-menu>
+          <v-row class>
+            <v-col cols="12" sm="6" md="3">
+              <v-select
+                id="hour-selector"
+                label="Hour"
+                :items="hours"
+                v-model="selectHour"
+                :disabled="!isFutureEffective"
+                filled
+              ></v-select>
+            </v-col>
+            <span :class="{ 'disabled': !isFutureEffective, 'time-colon': true }">:</span>
+            <v-col cols="12" sm="6" md="3">
+              <v-select
+                id="minute-selector"
+                label="Minute"
+                :items="minutes"
+                v-model="selectMinute"
+                :disabled="!isFutureEffective"
+                filled
+              ></v-select>
+            </v-col>
+            <v-col cols="12" sm="6" md="3">
+              <v-select
+                id="am-pm-selector"
+                :items="timePeriod"
+                v-model="selectPeriod"
+                :disabled="!isFutureEffective"
+                filled
+              ></v-select>
+            </v-col>
+            <v-col cols="12" sm="6" md="2" class="label-col">
+              <span class="time-zone-label" :class="{ 'disabled': !isFutureEffective }">Pacific Time</span>
+            </v-col>
+          </v-row>
+        </div>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Watch, Emit, Prop } from 'vue-property-decorator'
+import { Action } from 'vuex-class'
+
+// Mixins
+import { DateMixin } from '@/mixins'
+
+// Constants
+import { ISIMMEDIATE, ISFUTUREEFFECTIVE } from '@/constants'
```

Comment:
If these aren't used anywhere else then you could simply declare them as `readonly` properties in this module.

Reply:
For sure! Not used anywhere else at his time

Reply:
Are you ok if i leave them as constants? I'd prefer to have less properties on the component if i could. Referencing them also means more code as `this.property` opposed to `constant`. :)

Reply:
It's 6 of one, 1/2 dozen of the other...
As constants, you're polluting the app namespace instead of just your component, but yes, you can just use `constant`.
As properties, you have to use `this.constant` (which I don't mind personally), but then they're localized to just your property.
Up to you (author's prerogative)!

Reply:
Fair enough, thanks for the help on this. Food for thought!